client/attest.go

```diff
@@ -287,7 +292,15 @@ func (k *Key) Attest(opts AttestOpts) (*pb.Attestation, error) {
 	}
 	attestation.AkCert = k.CertDERBytes()
 	for _, sel := range sels {
-		quote, err := k.Quote(sel, opts.Nonce)
+		extraData := opts.Nonce
```

Comment:
this can be outside the loop, I think the nonce is only related to the key, not the pcr bank?